packages/main/test/specs/ComboBox.spec.js

```diff
@@ -768,7 +768,7 @@ describe("Keyboard navigation", async () => {
 		const pickerIcon = await comboBox.shadow$("[input-icon]");
 		const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#combo2");
 		const respPopover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
-		let listItem;
+		let listItem, inputValue;
```

Comment:
Can you remove the changes to the ComboBox spec? They seem to be added by mistake.